python-package/setup.py

```diff
@@ -246,7 +246,11 @@ def initialize_options(self) -> None:
         self.hdfs = False
         self.precompile = False
         self.time_costs = False
-        self.nomp = False
+        #Add for PASE on i
+        if system() == 'OS400':
```

Comment:
Please do this in `CMakeLists.txt` so most builds of LightGBM will benefit from it. The Python package shouldn't need to be aware of this.